app/src/main/java/org/simple/clinic/registration/facility/RegistrationFacilitySelectionScreenController.kt

```diff
@@ -46,8 +46,8 @@ class RegistrationFacilitySelectionScreenController @Inject constructor(
         .flatMap {
```

Comment:
This will require the type of `FacilityRepository.recordCount()` to be changed from `Single` -> `Observable`… which I've already done in an upcoming commit. I'll also do this in my next PR.

Reply:
👍